docs/source/architecture/permissioning_requirement.rst

```diff
@@ -1,33 +1,38 @@
-**************************
-Permissioning Requirements
-**************************
+********************
+Permissioning Design
+********************
 
-The following is a detailed list of permissioning requirements, capabilities,
-and user stories that aid in the explanation and discussion of the
-permissioning design for Hyperledger Sawtooth.
+Sawtooth includes the ability to control network, validator, and transactor
```

Comment:
We list three items here, but only 2 in the bullet points below.

Reply:
I'll remove "network," from this sentence.

Reply:
Done.